elastic/apm-server
# Collect aggregates from expvar metrices

## Motivation/summary

Collect aggregated metrics from the result of expvar. Currently collecting max, min and average.

## Checklist

~~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~~
~~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~~
~~- [ ] Documentation has been updated~~

## How to test these changes

1. Run the `apmtest` benchmark
2. Run apmbench test with `-detailed` flag
3. Observe the following fields in the output: `gc_cycles`, `max_rss`, `max_goroutines`, `max_heap_alloc`, `max_heap_objects` and `mean_available_indexers`



## Related issues

Closes #7817
